### PR TITLE
[WEB-2636] [WEB-2637] Extend functionality of AccessLogger

### PR DESCRIPTION
**Pull request by mshahidi**

### CODE REVIEW

File: src/AccessLogWriter.php [hunk 1]
```diff
@@ -57,20 +63,29 @@ public function log(?Request $request, Response $response): void
...
                 'geo_ip_info'       => $geo,
                 'client_app'        => $app,
                 'request_scopes'    => $request->getAttribute(RequestMiddleware::SCOPES, []),
-                'request_user_id'   => $request->getAttribute(RequestMiddleware::USER_ID, '')
+                'request_user_id'   => $request->getAttribute(RequestMiddleware::USER_ID, ''),
+                'extra'             => $extra
             ];
 
+            if ($response !== null) {
+                $date['http_status_code'] = $response->getStatusCode();
```
Comment (chris-die) [inline src/AccessLogWriter.php]: Typo: `$date` should be `$data`

File: src/AccessLogWriter.php [hunk 2]
```diff
@@ -57,20 +63,29 @@ public function log(?Request $request, Response $response): void
...
+                $seratoErrorCode = $response->getHeaderLine('X-Serato-ErrorCode');
+                if ($seratoErrorCode !== null && $seratoErrorCode !== '') {
+                    $data['serato_error_code'] = $seratoErrorCode;
+                }
+            }
+
             $this->logger->log(
                 $this->logLevel,
-                $request->getMethod() . ' ' . $request->getUri()->getPath(),
+                '',
```
Comment (chris-die) [inline src/AccessLogWriter.php]: Why has the request information been remove from here?

File: src/AccessLogWriter.php (hunk 1 above)
Comment (mshahidi) [reply src/AccessLogWriter.php]: Good catch.

File: src/AccessLogWriter.php (hunk 2 above)
Comment (mshahidi) [reply src/AccessLogWriter.php]: I moved it to the data array so that we can have all information in one single array.

**Pull request merged**